apps/cave/lib/env.conf.ts

```diff
@@ -20,7 +20,7 @@ export const NEXT_PUBLIC_CONCAVE_RPC_KEY = process.env.NEXT_PUBLIC_CONCAVE_RPC_K
 export const NEXT_PUBLIC_INFURA_ID = process.env.NEXT_PUBLIC_INFURA_ID
 export const NEXT_PUBLIC_ALCHEMY_ID = process.env.NEXT_PUBLIC_ALCHEMY_ID
 export const NEXT_PUBLIC_ETHERSCAN_API_KEY = process.env.NEXT_PUBLIC_ETHERSCAN_API_KEY
-
+export const NEXT_PUBLIC_LOCALHOST_RPC = process.env.NEXT_PUBLIC_LOCALHOST_RPC
```

Comment:
the port is hardcoded in the hardhat.config file, I don't think we need a env for this